pkg/types/service_instance.go

```diff
@@ -30,19 +30,24 @@ import (
 // ServiceInstance struct
 type ServiceInstance struct {
 	Base
-	Name            string               `json:"name"`
-	ServicePlanID   string               `json:"service_plan_id"`
-	PlatformID      string               `json:"platform_id"`
-	DashboardURL    string               `json:"dashboard_url,omitempty"`
-	MaintenanceInfo json.RawMessage      `json:"maintenance_info,omitempty"`
-	Context         json.RawMessage      `json:"context,omitempty"`
-	UpdateValues    InstanceUpdateValues `json:"-"`
-	PreviousValues  json.RawMessage      `json:"-"`
+	Name                 string               `json:"name"`
+	ServicePlanID        string               `json:"service_plan_id"`
+	ReferencedInstanceID string               `json:"referenced_instance_id"`
+	PlatformID           string               `json:"platform_id"`
+	DashboardURL         string               `json:"dashboard_url,omitempty"`
+	MaintenanceInfo      json.RawMessage      `json:"maintenance_info,omitempty"`
+	Context              json.RawMessage      `json:"context,omitempty"`
+	UpdateValues         InstanceUpdateValues `json:"-"`
+	PreviousValues       json.RawMessage      `json:"-"`
 
 	Parameters map[string]interface{} `json:"parameters,omitempty"`
 	Usable     bool                   `json:"usable"`
+	Shareable  bool                   `json:"shareable"`
```

Comment:
why do we need this Shareable field on the instance? isn't it a property of the plan and not the instance?

Reply:
Hey Eyal, 

This property is mainly for the cockpit and the CLIs users, an indication of whether the instance is can be shared or not.

We set this property on the instance when is created from a shareable plan.

Reply:
TODO
No need for the shareable property on the instance
Inform Avi about the change

Reply:
done